x/mint/abci.go

```diff
@@ -20,7 +20,11 @@ func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
 	// recalculate inflation rate
 	totalStakingSupply := k.StakingTokenSupply(ctx)
 	bondedRatio := k.BondedRatio(ctx)
-	minter.Inflation = minter.NextInflationRate(params, bondedRatio)
+	if ic == nil {
+		minter.Inflation = minter.NextInflationRate(params, bondedRatio)
```

Comment:
Can `minter.NextInflationRate()` be turned into a `types.InflationCalculationFn` function so that it can be the default?

Reply:
Done